src/Datadog.Trace/Configuration/GlobalSettings.cs

```diff
@@ -184,5 +223,21 @@ private static bool TryLoadHostingEnvironmentPath(out string hostingPath)
             hostingPath = default;
             return false;
         }
+
+        private static void LoadVendorFactoryConfigurator()
+        {
+            if (Source.VendorPluginLoaded)
```

Comment:
Overleft `if`, will be fixed later.

Reply:
`VendorPluginLoaded` is not needed as well. Correct?